### PR TITLE
fix typos for fmri-preprocessing-tutorial.md

### PR DESCRIPTION
**Pull request by bendichter**

### CODE REVIEW

File: docs/tutorial/fmri-preprocessing-tutorial.md [hunk 1]
```diff
@@ -101,9 +100,7 @@ Once you're happy with the surfaces, you can move onto running Freesurfer!
     * For input, select the staged preprocessed, 'acpc_aligned'-tagged anatomical (T1w & T2w) images by clicking the drop-down menu and finding the appropriate datasets.
     * Select the boxes for 'hippocampal' and 'thalamicnuclei'. Leave all other defaults the same
     ![Free Surfer Options](../img/fmri_prep_tutorial_free_surfer_options.png)
-
     ![Free Surfer Options 1](../img/fmri_prep_tutorial_free_surfer_options_1.png)
-
```
Comment (bendichter) [inline docs/tutorial/fmri-preprocessing-tutorial.md:106]: Attempting to fix this formatting error:

image

**Review by francopestilli — CHANGES_REQUESTED** [review]
thanks @bendichter a few comments and changes @anibalsolon

File: docs/tutorial/fmri-preprocessing-tutorial.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ This tutorial will use a combination of skills developed in the [Introduction tu
...
+The first step of diffusion preprocessing often involves processing the anatomical images. In order to guarantee that any generalizations regarding location made from the preprocessed diffusion data is anatomically-informed, we must have both of our anatomical (T1w or T2w) images and our diffusion MRI images **aligned**. One way we can make this easier for [fMRIPrep](https://brainlife.io/app/5c61c69f14027a01b14adcb3) is by aligning the anatomical images in such a way that the center of the brain is centered in the image. We refer to this as **ACPC-aligned**, as we are aligning the data to the **anterior commissure-posterior comissure plane**. This is the first step in fMRI preprocessing, and it is typically done with the [FSL Anat (T1)](https://brainlife.io/app/5e3c87ae9362b7166cf9c7f4) app. Once we've centered our anatomical image, we can move onto functional MRI preprocessing.
```
[the remark this answers is not in the document]
Comment (bendichter) [reply docs/tutorial/fmri-preprocessing-tutorial.md:12]: The name of the app is "(T1)". Maybe it should be "(T1w)", but my goal here is to ensure that the name in the tutorial matches the name of the app.

[the remark this answers is not in the document]
Comment (anibalsolon) [reply docs/tutorial/fmri-preprocessing-tutorial.md:12]: Better keep as T1 indeed!

**Pull request merged**